### DISCUSSION TITLE
Hi all! I have a doubt about the slots in the calendar

### DISCUSSION THREAD

**Discussion by gr4v3** [vkurko/calendar#272] [Q&A]
![image](https://github.com/vkurko/calendar/assets/227474/9f0bbe69-c7a3-48f6-a5ba-262aa9653ffa)
Currently it is only hiding 2 slots per day which results in showing the +2 more as shown in the screenshot above. 

I wanted to ask if there is a way to hide all the events of the day and like that the counter would increase also. 

So per example: a day with 5 events it would simply show +5 events and hide all the events or could be like for 5 events show at least 1 and say +4 events.

**Comment by vkurko** [comment]
Hello,

It looks like you need a version of the [dayMaxEvents](https://github.com/vkurko/calendar?tab=readme-ov-file#daymaxevents) option that takes a numeric value like in [FullCalendar](https://fullcalendar.io/docs/dayMaxEvents).

Unfortunately, the current version of this option in Event Calendar only accepts `true`/`false` values. In the future, the ability to specify a numeric value will be implemented.
